app/grants/views.py

```diff
@@ -442,16 +446,17 @@ def get_interrupted_contributions(request):
 
 def get_grants(request):
     grants = []
-    _grants = None
+    collections = []
```

Comment:
@thelostone-mc I was thinking maybe we can move collection to his own endpoint to make the code simpler, lets chat about it.

Reply:
Def yes ! Do we wanna do that though ?